PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -0,0 +1,11 @@
+
+## Version m.n.o
+_Date TBD_
+
+### Table changes
+* Add practice type to the test functionality table
+
+### Data modifications
+* Updated 2014 functionality test records to specify their associated practice type
+
+---
```

Comment:
I think there needs to be a blank line after line 11. Otherwise, during deployment, the `---` will butt up against the "previous" release notes section